Lib/defcon/tools/notifications.py

```diff
@@ -373,31 +375,22 @@ def __init__(self):
         self._order = []
 
     def keys(self):
-        return list(self._order)
+        return iter(self)
 
     def values(self):
-        return [self[key] for key in self]
+        for key in self:
+            yield self[key]
 
     def items(self):
-        return [(key, self[key]) for key in self]
+        for key in self:
+            yield (key, self[key])
 
     def __iter__(self):
         order = self._order
         while order:
             yield order[0]
             order = order[1:]
 
-    def iterkeys(self):
-        return iter(self)
-
-    def itervalues(self):
-        for key in self:
-            yield self[key]
-
-    def iteritems(self):
-        for key in self:
-            yield (key, self[key])
-
```

Comment:
(and revert changes to keys, values, items). The changes should be kept for Python 3 so to match other collection APIs (e.g., for 3.5+ we use the stdlib OrderedDict)